### DISCUSSION TITLE
what is "Scheduler > Jobs > Run Log"?

### DISCUSSION THREAD

**Discussion by majkyyx** [dbeaver/dbeaver#12950] [Q&A]
Hi, i'm using oracle database (version doesn' matter).

I thought "Run Log" is for display data from runs of the job, but this is showing only "No Data", here is screenshot:
![image](https://user-images.githubusercontent.com/14957734/123061127-a4d3ba00-d40b-11eb-9cbf-9b22ccdbcc23.png)

In oracle is view:
```sql
SELECT * FROM USER_SCHEDULER_JOB_RUN_DETAILS WHERE job_name = 'WF_CHECKDEADLINES'
```
![image](https://user-images.githubusercontent.com/14957734/123061238-bfa62e80-d40b-11eb-8850-44ac9b25a50b.png)

It's a bug or my bad assumption of using run log ? 

thx a lot for any help ;-)

**Comment by flambchap** [comment]
Hi @majkyyx. Unfortunately, I can not reproduce your situation. Could you please execute the following statement? It may help me to figure out what is going on. Thanks.

```sql
SELECT * FROM ALL_SCHEDULER_JOB_RUN_DETAILS WHERE job_name = 'WF_CHECKDEADLINES'
```

**Reply by majkyyx** [reply]
Hi Nikita, this statement work's too, it same result like USER_SCHEDULER_JOB_RUN_DETAILS, but there are all users from SID.

![image](https://user-images.githubusercontent.com/14957734/123095231-26d4da80-d42e-11eb-8357-375985a96af2.png)

Do you know which statement is used for "Run Log" ?

**Reply by flambchap** [reply]
The one I asked you to execute :) FYI, you can see which statements DBeaver executes in `Window` -> `Show view` -> `Query manager`. 

To be quite honest, I don't have any good ideas at the moment. Hope I'll figure out something soon.

**Reply by majkyyx** [reply]
Do you see any rusults of run log? can you give screenshot of some job ?

I tried enable "meta" and "meta dll" in query manager, there are two queries:

```sql
SELECT * FROM ALL_SCHEDULER_JOB_ARGS WHERE OWNER='VERSO2_02' AND JOB_NAME='WF_CHECKDEADLINES' ORDER BY ARGUMENT_POSITION
```
```sql
SELECT * FROM ALL_SCHEDULER_JOBS WHERE OWNER='VERSO2_02' ORDER BY JOB_NAME
```
both are with result "Success" and  working in normal sql window.. 

maybee i need diferent odbc driver or something..

**Reply by majkyyx** [reply]
I found error in Error Log, if i click on Run Log, i can see this in error log:

``Cannot determine transaction isolation level due to connection hanging. Setting to NONE.``
``An exception stack trace is not available.``
